cli/src/ui.rs

```diff
@@ -92,7 +92,10 @@ impl UiOutput {
             scroll_past_eof: false,
             ..Default::default()
         };
-        let mut pager = streampager::Pager::new_using_stdio_with_config(streampager_config)?;
+        // Initialize with tty instead of stdio. Pager is allowed so long as
+        // stdout is a tty. stdin may be redirected.
+        let mut pager =
+            streampager::Pager::new_using_system_terminal_with_config(streampager_config)?;
 
```

Comment:
(Optional) I'm also leaning towards having a CHANGELOG entry for this. Some people may have run into this bug and might appreciate being informed about its fix. But, ultimately, up to you, this is a bit of a borderline case.

Reply:
Added changelog entry. It will probably work on Windows since tty can still be detected by using stdout.